### PR TITLE
[UXIT-1782] Make Sponsor link required

### PR DESCRIPTION
**Pull request by barbaraperic**

### CODE REVIEW

**Review by CharlyMartin — CHANGES_REQUESTED** [review]
Hi @barbaraperic 

Shouldn’t we also update the `SponsorsSchema` Zod schema to match the CMS configuration?

```ts
const sponsorSchema = z.object({
  name: z.string(),
  website: z.string().url(), // Remove .optional()
  image: ImagePropsSchema,
})
```

And If we do, we might need to backfill the data for sponsors with missing website links (I haven’t verified if any are missing)

**Comment by barbaraperic** [comment]
> Hi @barbaraperic
> 
> Shouldn’t we also update the `SponsorsSchema` Zod schema to match the CMS configuration?
> 
> ```ts
> const sponsorSchema = z.object({
>   name: z.string(),
>   website: z.string().url(), // Remove .optional()
>   image: ImagePropsSchema,
> })
> ```
> 
> And If we do, we might need to backfill the data for sponsors with missing website links (I haven’t verified if any are missing)

Great points, fixed!

File: public/admin/config.yml [hunk 1]
```diff
@@ -93,7 +93,7 @@ event_sponsor_config: &event_sponsor_config
   - name: "website"
     label: "website"
     widget: "string"
-    required: false
+    required: true
```
Comment (mirhamasala) [inline public/admin/config.yml:96]: This is the default.

**Review by mirhamasala — APPROVED** [review]
@barbaraperic - I removed [the broken link](https://filecoinfoundation.slack.com/archives/C06MTEQ3SP6/p1732859938176049). Also `required: true` is the default so removed that as well. Good to go.

**Pull request merged**